conda/conda-build/meta.yaml

```diff
@@ -177,13 +178,14 @@ requirements:
 {% endif %}
 {% if ucx_configured_bool %}
     - ucx >=1.14
+    - openmpi
```

Comment:
We would have probably caught this if we were testing our conda build. In the current build process, we have openmpi from the env generation script during the build, but it was not present in the package requirements.